lisa/sut_orchestrator/hyperv/platform_.py

```diff
@@ -0,0 +1,323 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import random
+import string
+from pathlib import PurePath, PureWindowsPath
+from typing import Any, List, Optional, Type, cast
+
+from lisa import feature, schema, search_space
+from lisa.environment import Environment
+from lisa.node import RemoteNode
+from lisa.platform_ import Platform
+from lisa.tools import Cp, HyperV, Ls, Mkdir, PowerShell, Unzip
+from lisa.util.logger import Logger, get_logger
+
+from .. import HYPERV
+from .context import get_node_context
+from .schema import HypervNodeSchema, HypervPlatformSchema
+from .serial_console import SerialConsole, SerialConsoleLogger
+
+
+class _HostCapabilities:
+    def __init__(self) -> None:
+        self.core_count = 0
+        self.free_memory_mib = 0
+
+
+class HypervPlatform(Platform):
+    def __init__(
+        self,
+        runbook: schema.Platform,
+    ) -> None:
+        super().__init__(runbook=runbook)
+
+    @classmethod
+    def type_name(cls) -> str:
+        return HYPERV
+
+    @classmethod
+    def supported_features(cls) -> List[Type[feature.Feature]]:
+        return [SerialConsole]
+
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        self._hyperv_runbook = self._get_hyperv_runbook()
+        self.server_node = self._initialize_server_node()
```

Comment:
It's used in platform only, so it should be `_server`

Reply:
changed.